### PR TITLE
Create secrets panel is too short (hard to select last SA)

### PR DESCRIPTION
**Pull request by carlos-logro**
issue: #400 

# Changes
Managed to make the Service Account dropdown easier to access by adding a css style to it so it appears upward instead of downward.

### CODE REVIEW

**Comment by carlos-logro** [comment]
Problem:
Screen Shot 2019-08-06 at 1 47 29 PM

**Comment by carlos-logro** [comment]
Solution: 
Screen Shot 2019-08-06 at 1 47 11 PM

**Comment by AlanGreene** [comment]
If we used a [Select](http://react.carbondesignsystem.com/?path=/story/select--default) (technically more correct for use in a form) here instead of a Dropdown it shouldn't have the same issue as it uses the native browser select menu which should be able to extend over the edge of the dialog.

**Comment by carlos-logro** [comment]
> If we used a [Select](http://react.carbondesignsystem.com/?path=/story/select--default) (technically more correct for use in a form) here instead of a Dropdown it shouldn't have the same issue as it uses the native browser select menu which should be able to extend over the edge of the dialog.

that means refactoring the service account dropdown component??

**Comment by carlos-logro** [comment]
@AlanGreene 

didn't refactor the sa component but i did changed the one in the modal which meant using action & reducers to get the sa in the store!

Screen Shot 2019-08-07 at 3 51 19 PM
Screen Shot 2019-08-07 at 3 51 27 PM

File: src/components/SecretsModal/BasicAuthFields.js [hunk 1]
```diff
@@ -24,8 +23,13 @@ const BasicAuthFields = props => {
     handleChangeTextInput,
     handleChangeServiceAccount,
     invalidFields,
-    serviceAccount
+    serviceAccounts
   } = props;
+
+  const saItems = serviceAccounts
+    .filter(sa => sa.metadata.namespace === namespace)
+    .map(sa => <SelectItem value={sa.metadata.name} text={sa.metadata.name} />);
```
Comment (AlanGreene) [inline src/components/SecretsModal/BasicAuthFields.js]: missing `key` prop

Comment (carlos-logro) [reply src/components/SecretsModal/BasicAuthFields.js]: ill add a style to hide it then.

**Comment by carlos-logro** [comment]
@AlanGreene hid the chevron icon! When you can please give the PR another review, Thank you

**Pull request merged**